src/transformers/models/videomae/modeling_videomae.py

```diff
@@ -859,6 +863,10 @@ def forward(
                     tubelet_size * patch_size * patch_size * num_channels,
                 )
             else:
+                if self.config.num_channels != 3:
+                    raise ValueError(
+                        "Can't unnormalize non-RGB images. Consider setting config.norm_pix_loss to False."
```

Comment:
I feel a bit confused here. We are in  the `else` block, which means we are already `self.config.norm_pix_loss == False`..
But the message says `Consider setting config.norm_pix_loss to False. ...`. Also this block doesn't have normalization, and the block  `if self.config.norm_pix_loss:` has `# step 4: normalize`.

@NielsRogge  @layjain  Could you double check?